container_registry/container_analysis/create_occurrence.go

```diff
@@ -41,16 +40,27 @@ func createOccurrence(resourceURL, noteID, occProjectID, noteProjectID string) (
 		Occurrence: &grafeaspb.Occurrence{
 			NoteName: fmt.Sprintf("projects/%s/notes/%s", noteProjectID, noteID),
 			// Attach the occurrence to the associated resource uri.
-			Resource: &grafeaspb.Resource{
-				Uri: resourceURL,
-			},
+			ResourceUri: resourceURL,
 			// Details about the vulnerability instance can be added here.
 			Details: &grafeaspb.Occurrence_Vulnerability{
-				Vulnerability: &vulnerability.Details{},
+				Vulnerability: &grafeaspb.VulnerabilityOccurrence{
+					PackageIssue: []*grafeaspb.VulnerabilityOccurrence_PackageIssue{
+						&grafeaspb.VulnerabilityOccurrence_PackageIssue{
+							AffectedCpeUri: "your-uri-here",
+							AffectedPackage: "your-package-here",
+							MinAffectedVersion: &grafeaspb.Version{
+								Kind: grafeaspb.Version_MINIMUM,
+							},
+							FixedVersion: &grafeaspb.Version{
+								Kind: grafeaspb.Version_MAXIMUM,
+							},
+						},
+					},
+				},
```

Comment:
FYI @Katee : It looks like the minimal Note and Occurrence has got a lot more complicated in v1 compared to beta1. Just wanted to make sure that was intentional (and let me know if this looks like a good sample, or if you have another idea in mind)